src/ripple/protocol/impl/Feature.cpp

```diff
@@ -450,6 +450,7 @@ REGISTER_FEATURE(NonFungibleTokensV1_1,         Supported::yes, DefaultVote::no)
 REGISTER_FIX    (fixTrustLinesToSelf,           Supported::yes, DefaultVote::no);
 REGISTER_FIX    (fixRemoveNFTokenAutoTrustLine, Supported::yes, DefaultVote::yes);
 REGISTER_FEATURE(ImmediateOfferKilled,          Supported::yes, DefaultVote::no);
+REGISTER_FIX    (fixUniversalNumber,            Supported::yes, DefaultVote::yes);
```

Comment:
Does this need to be `DefaultVote::yes`? 

Unless the fix is critical, we prefer to keep the default vote at `no` for the 1st release that understands the amendment. Subsequent releases can have the default vote at `yes`. This has the effect of making it more likely that node operators have more lead time in order to review the changes and update their rippled software versions.

Reply:
Changed to `no` in https://github.com/XRPLF/rippled/pull/4414. We may change it back to `yes` in a future release, but not in 1.10.